### DISCUSSION TITLE
Trouble running on MacOS

### DISCUSSION THREAD

**Discussion by etabackman** [lunasec-io/lunasec#373] [Help/Support] [answered]
After extracting to `/usr/local/bin` as explained here: https://github.com/lunasec-io/lunasec/discussions/339

I am getting `command not found` when trying to run this on macOS:

`sudo ./log4shell_1.4.1-log4shell_Darwin_x86_64 scan /my/path`

or this:

`sudo log4shell_1.4.1-log4shell_Darwin_x86_64 scan /my/path`

both from within `/usr/local/bin`.

if I run without `sudo` I get this output:

`Permission denied`

Will someone help? Thanks!

**Comment by SW-41** [comment]
having the same problem

**Comment by freeqaz** [comment]
We're taking a look at this right now -- we rented a Mac in the cloud to help figure out why this is broken. I'm guessing it's a PATH issue, but I'll have to verify.

As a workaround, you should just be able to execute the CLI in the current folder `./log4shell_1.4.1-log4shell_Darwin_x86_64`. That'll get around pathing issue, but it won't be installed globally (at that point)

**Comment by factoidforrest** [comment, marked as answer]
I am updating the blog post with better instructions.  You need to:
Copy the tool to /usr/local/bin
Ideally rename it to log4shell
chmod +x it so that its executable.  That's it but I could see how that would be confusing. 

```
cp ~/Downloads/log4shell_1.4.1-log4shell_Darwin_x86_64 /usr/local/bin/log4shell
chmod +x /usr/local/bin/log4shell
log4shell scan your-project-dir
```
